zbus/tests/basic.rs

```diff
@@ -117,6 +117,24 @@ async fn fdpass_systemd_async() {
     f.metadata().unwrap();
 }
 
+#[cfg(all(target_os = "linux", feature = "ibus"))]
+#[test]
+fn ibus() {
+    use zbus::{blocking, conn};
+
+    let ibus_blocking = blocking::connection::Builder::ibus()
+        .unwrap()
+        .build()
+        .unwrap();
+    let ibus_non_blocking = block_on(block_on(conn::Builder::ibus()).unwrap().build()).unwrap();
+
+    assert!(!ibus_blocking.server_guid().is_empty());
+    assert_eq!(
+        ibus_blocking.server_guid().is_empty(),
+        ibus_non_blocking.server_guid().is_empty()
+    );
```

Comment:
comparing booleans is always confusing. :) This statement can be removed as well for the same reason mentioned above.

Reply:
As I wrote before, fixups to existing commits should go into those commits. So after making the changes, you need something like `git commit --fixup HASH_OF_TARGET_COMMIT && git rebase --autosquash -i HASH_OF_TARGET_COMMIT^` (notice the use of `^` to refer to the parent commit).

Reply:
after that `git push --force`.